### ISSUE TITLE
google_mobile_ads (from `.symlinks/plugins/google_mobile_ads/ios`) was resolved to 0.0.1, which depends on Google-Mobile-Ads-SDK (= 8.13.0) was resolved to 8.13.0, which depends on GoogleAppMeasurement (< 9.0, >= 7.0)

### ISSUE THREAD

**Issue by hukusuke1007** [googleads/googleads-mobile-flutter#594]
## Plugin Version

google_mobile_ads: ^1.3.0

## Steps to Reproduce

pubspec.yaml

```
  firebase_core: ^1.19.1
  firebase_analytics: ^9.1.10
  cloud_firestore: ^3.2.1
  firebase_auth: ^3.4.1
  firebase_storage: ^10.3.1
  cloud_functions: ^3.3.1
  firebase_messaging: ^11.4.4
  firebase_crashlytics: ^2.8.4
  firebase_dynamic_links: ^4.3.1

  google_mobile_ads: ^1.3.0
```

1. Run `pod install`.

**Expected results:** 
Non error.

**Actual results:** 
Error occurred.

```
[!] CocoaPods could not find compatible versions for pod "GoogleAppMeasurement":
  In Podfile:
    firebase_analytics (from `.symlinks/plugins/firebase_analytics/ios`) was resolved to 9.1.12, which depends on
      Firebase/Analytics (= 9.2.0) was resolved to 9.2.0, which depends on
        Firebase/Core (= 9.2.0) was resolved to 9.2.0, which depends on
          FirebaseAnalytics (~> 9.2.0) was resolved to 9.2.0, which depends on
            FirebaseAnalytics/AdIdSupport (= 9.2.0) was resolved to 9.2.0, which depends on
              GoogleAppMeasurement (= 9.2.0)

    google_mobile_ads (from `.symlinks/plugins/google_mobile_ads/ios`) was resolved to 0.0.1, which depends on
      Google-Mobile-Ads-SDK (= 8.13.0) was resolved to 8.13.0, which depends on
        GoogleAppMeasurement (< 9.0, >= 7.0)

```
<details>
  <summary>Logs</summary>

```
```

```
```

```
```

</details>

**Comment by jonafeucht** [comment]
As a workaround you can use the fork from @jjliu15, who has also already opened a pull request #590.

```
  google_mobile_ads:
    git:
      url: https://github.com/jjliu15/googleads-mobile-flutter-1.git
      ref: update_version
      path: packages/google_mobile_ads/
```

**Comment by marcellocamara** [comment]
Same problem here. I think I'll wait to merge this mentioned PR.
Just don't know how long it will take 😕

**Comment by StevenTsuii** [comment]
anyone can fix this issue...????

**Comment by GENYF** [comment]
I've tried many ways, but this issue is not resolved.

My development environment is 'Flutter SDK 3.0.3', 'Dart SDK 2.17.5', 'MacBook Pro 13 (M1, 2020)', and 'MacOS Monterey 12.4'.

**Comment by StevenTsuii** [comment]
Please help.....!!!

**Comment by maheshmnj** [comment]
Hi @hukusuke1007, Thanks for filing the issue. I am able to reproduce the issue with the new release of `google_mobile_ads: ^1.3.0`. 


```yaml
dependencies:
  firebase_core: ^1.19.1
  firebase_analytics: ^9.1.10
  cloud_firestore: ^3.2.1
  firebase_auth: ^3.4.1
  firebase_storage: ^10.3.1
  cloud_functions: ^3.3.1
  firebase_messaging: ^11.4.4
  firebase_crashlytics: ^2.8.4
  firebase_dynamic_links: ^4.3.1

  google_mobile_ads: ^1.3.0
```

<details>
<summary>logs</summary>

```
mahesh@Maheshs-MacBook-Air-M1 ios % arch -x86_64 pod install    
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Analyzing dependencies
cloud_firestore: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
cloud_functions: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_analytics: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_auth: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_core: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
Ignoring ffi-1.15.0 because its extensions are not built. Try: gem pristine ffi --version 1.15.0
Ignoring ffi-1.14.2 because its extensions are not built. Try: gem pristine ffi --version 1.14.2
Warning: firebase_app_id_file.json file does not exist. This may cause issues in upload-symbols. If this error is unexpected, try running flutterfire configure again.
firebase_crashlytics: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_dynamic_links: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_messaging: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
firebase_storage: Using Firebase SDK version '9.2.0' defined in 'firebase_core'
[!] CocoaPods could not find compatible versions for pod "GoogleAppMeasurement":
  In snapshot (Podfile.lock):
    GoogleAppMeasurement (< 9.0, = 8.15.0, >= 7.0)

  In Podfile:
    firebase_analytics (from `.symlinks/plugins/firebase_analytics/ios`) was resolved to 9.1.12, which depends on
      Firebase/Analytics (= 9.2.0) was resolved to 9.2.0, which depends on
        Firebase/Core (= 9.2.0) was resolved to 9.2.0, which depends on
          FirebaseAnalytics (~> 9.2.0) was resolved to 9.2.0, which depends on
            FirebaseAnalytics/AdIdSupport (= 9.2.0) was resolved to 9.2.0, which depends on
              GoogleAppMeasurement (= 9.2.0)

    google_mobile_ads (from `.symlinks/plugins/google_mobile_ads/ios`) was resolved to 0.0.1, which depends on
      Google-Mobile-Ads-SDK (= 8.13.0) was resolved to 8.13.0, which depends on
        GoogleAppMeasurement (< 9.0, >= 7.0)


You have either:
 * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
 * changed the constraints of dependency `GoogleAppMeasurement` inside your development pod `firebase_analytics`.
   You should run `pod update GoogleAppMeasurement` to apply changes you've made.
```
</details> 

cc: @jjliu15

**Comment by Omar-Salem** [comment]
same issue

**Comment by hemal-moradiya** [comment]
Is there any workaround for this issue?

**Comment by hemal-moradiya** [comment]
> As a workaround you can use the fork from @jjliu15, who has also already opened a pull request #590.
> 
> ```
>   google_mobile_ads:
>     git:
>       url: https://github.com/jjliu15/googleads-mobile-flutter-1.git
>       ref: update_version
>       path: packages/google_mobile_ads/
> ```
> 
> Reference: #591

I have tried this one but it didn't work with Flutter version 3.0.2

**Comment by kevin76g** [comment]
> As a workaround you can use the fork from @jjliu15, who has also already opened a pull request #590.
> 
> ```
>   google_mobile_ads:
>     git:
>       url: https://github.com/jjliu15/googleads-mobile-flutter-1.git
>       ref: update_version
>       path: packages/google_mobile_ads/
> ```
> 
> Reference: #591

Same issue here. This worked for me with Flutter version 3.0.4 and MacOS 12.4.

**Comment by GENYF** [comment]
I found a temporary solution.

`Firebase iOS SDK` version changed to `9.2.0` due to `firebase_core` `1.9.0` update.

This is presumed to result in an error. In my case, I solved it by downgrading to `firebase_core` `1.8.0.`

`google_mobile_ads` `1.3.0` is available, and in my case, iOS build was successful.

In addition, if you are using a version of `firebase_storage` `10.3.0` or later, you will need to downgrade to `firebase_storage` `10.2.18`.

However, since this is a temporary solution, we have to wait for the merger of PR #590.

**Comment by Waheed388** [comment]
The culprit is firebase_analytics. The moment I removed this package, the code is working fine.

**Comment by nzackoya** [comment]
Every time same thing happens with firebase libs

**Comment by afine23** [comment]
> google_mobile_ads:

How can I install the fork?

**Comment by mono0926** [comment]
> https://github.com/googleads/googleads-mobile-flutter/issues/594#issuecomment-1172875150

#590 was merged and `update_version` branch of https://github.com/jjliu15/googleads-mobile-flutter-1 has been deleted. So, current workaround is here:

```yaml
  google_mobile_ads:
    git:
      url: https://github.com/googleads/googleads-mobile-flutter
      ref: 356875362ddb2ac2e0056385d2729c093722fb8d
      path: packages/google_mobile_ads
```

**Comment by Cubel89** [comment]
I have the same problem. I hope it is resolved soon

**Comment by Charlinjoeaht** [comment]
This is really blocking my activities. Please resolve it as soon as possible

**Comment by hhfa1990** [comment]
Same error with this versions:

```
firebase_core: ^1.19.2
firebase_analytics: ^9.2.0
google_mobile_ads: ^1.3.0
```

**Comment by westlinkin** [comment]
need an update here!

**Comment by mk-dev-1** [comment]
Come one guys, all you need is to stop spamming here! Everyone who is subscribed to this thread is receiving your pointless 'same here' comments. They contribute nothing to resolve the issue and won't help getting it resolved any faster. That being said, the below resolves the issue. All that is missing is an updated release.

> ```yaml
>   google_mobile_ads:
>     git:
>       url: https://github.com/googleads/googleads-mobile-flutter
>       ref: 356875362ddb2ac2e0056385d2729c093722fb8d
>       path: packages/google_mobile_ads
> ```

**Comment by mauriciotogneri** [comment]
@mk-dev-1 Can you elaborate more on how to apply that fix? Where do we have to put that? In what file? Thanks

**Comment by GoldenSoju** [comment]
@mauriciotogneri 
Replace the dependency for google_mobile_ads in your pubspec.yaml

```yaml
  google_mobile_ads: 1.3.0 #or what version number you are using
```
should be changed to:

```yaml
  google_mobile_ads:
    git:
      url: https://github.com/googleads/googleads-mobile-flutter
      ref: 356875362ddb2ac2e0056385d2729c093722fb8d
      path: packages/google_mobile_ads
```

**Comment by mauriciotogneri** [comment]
@GoldenSoju Thanks! I didn't know you could specify dependencies like that

**Comment by hicnar** [comment]
@mk-dev-1 Don't you think it would be fair if whoever controls this library released a statement saying what is holding back the release containing the fix and when is it likely to happen? Till then... SAME HERE!

**Comment by bigbang489** [comment]
I have the same issue

**Comment by isiyemijoe** [comment]
Still having the same issue

**Comment by asm495** [comment]
Just a comment to say that I am experiencing this too. I am having to manually modify my pubspec.yaml to build iOS and Android differently as per the guidance. Would be good to get this resolved.

**Comment by ritesh24** [comment]
I am experiencing this too. When the lib will be released on pub.dev ?

**Comment by billachusim** [comment]
Currently applying this popular fix and it seems it needs some time during pub get. large data? I will revert if it worked for me.
Flutter 3.05 google_mobile_ads: 1.3.0

> @mauriciotogneri Replace the dependency for google_mobile_ads in your pubspec.yaml
> 
> ```yaml
>   google_mobile_ads: 1.3.0 #or what version number you are using
> ```
> 
> should be changed to:
> 
> ```yaml
>   google_mobile_ads:
>     git:
>       url: https://github.com/googleads/googleads-mobile-flutter
>       ref: 356875362ddb2ac2e0056385d2729c093722fb8d
>       path: packages/google_mobile_ads
> ```
> 
> For reference: [Official Docs](https://dart.dev/tools/pub/dependencies#hosted-packages)

**Comment by billachusim** [comment]
> Currently applying this popular fix and it seems it needs some time during pub get. large data? I will report if it worked for me. Flutter 3.05 google_mobile_ads: 1.3.0
> 
> > @mauriciotogneri Replace the dependency for google_mobile_ads in your pubspec.yaml
> > ```yaml
> >   google_mobile_ads: 1.3.0 #or what version number you are using
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > should be changed to:
> > ```yaml
> >   google_mobile_ads:
> >     git:
> >       url: https://github.com/googleads/googleads-mobile-flutter
> >       ref: 356875362ddb2ac2e0056385d2729c093722fb8d
> >       path: packages/google_mobile_ads
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > For reference: [Official Docs](https://dart.dev/tools/pub/dependencies#hosted-packages)

It worked!!!
But you might have to fix few more bearable build time errors  after applying this.
In my case I was still faced with target output and input issue but i fixed that by deleting the existing ones in CP Pods Frameworks in build phases and let the build readd them.

**Comment by zmeggyesi** [comment]
@jjliu15 Who would be the person to run a new release of the plugin? This version mismatch seems to be causing a lot of headache to a lot of people.

**Comment by almogtovim** [comment]
> > Currently applying this popular fix and it seems it needs some time during pub get. large data? I will report if it worked for me. Flutter 3.05 google_mobile_ads: 1.3.0
> > > @mauriciotogneri Replace the dependency for google_mobile_ads in your pubspec.yaml
> > > ```yaml
> > >   google_mobile_ads: 1.3.0 #or what version number you are using
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > should be changed to:
> > > ```yaml
> > >   google_mobile_ads:
> > >     git:
> > >       url: https://github.com/googleads/googleads-mobile-flutter
> > >       ref: 356875362ddb2ac2e0056385d2729c093722fb8d
> > >       path: packages/google_mobile_ads
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > For reference: [Official Docs](https://dart.dev/tools/pub/dependencies#hosted-packages)
> 
> It worked!!! But you might have to fix few more bearable build time errors after applying this. In my case I was still faced with target output and input issue but i fixed that by deleting the existing ones in CP Pods Frameworks in build phases and let the build readd them.

Hey can you share more details?

**Comment by jjliu15** [comment]
This issue should be resolved in version 2.0.0 of the plugin, which updates the iOS GMA dependency to 9+

**Comment by rsorianosanchis** [comment]
> This issue should be resolved in version 2.0.0 of the plugin, which updates the iOS GMA dependency to 9+

I'm with the latest stable settings and I still need the patch on google_mobile_ads (2.0.0 not works for me). (Macbook air M1 2020). Many thanks to the comrades who shared this temporary solution in pubspec.yaml.
My settings:
<img width="585" alt="Captura de Pantalla 2022-08-04 a las 9 25 04" src="https://user-images.githubusercontent.com/43029933/182788887-74a6579c-66e9-4314-984a-3d2ca57e8aa0.png">

**Comment by KuyaKoya** [comment]
I downgraded to the following versions for now and this resolved my issue with the dependencies:
`firebase_analytics: ^8.2.0
firebase_core: ^1.5.0
firebase_messaging: ^10.0.0
google_mobile_ads: ^2.0.0`

**Comment by QuentinCG** [comment]
Sorry to re-open this subject but it seems to re-appear:

I get this issue with IOS and none of the following answers fix it (`Flutter 3.7.7` - latest stable at the time of this writing, `google_mobile_ads: ^2.0.0`):
```sh
    [!] CocoaPods could not find compatible versions for pod "Google-Mobile-Ads-SDK":
      In Podfile:
        google_mobile_ads (from `.symlinks/plugins/google_mobile_ads/ios`) was resolved to 0.0.1, which depends on
          Google-Mobile-Ads-SDK (~> 9.13)
    None of your spec sources contain a spec satisfying the dependency: `Google-Mobile-Ads-SDK (~> 9.13)`.
    You have either:
     * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
     * mistyped the name or version.
     * not added the source repo that hosts the Podspec to your Podfile.
```

I tried all this without success:
   - `flutter clean`
   - `rm -rf ios/Pods/ || true`
   - `rm ios/Podfile.lock || true`
   - `rm -rf "${HOME}/Library/Caches/CocoaPods" || true`
   - `cd ios/ && pod setup && pod update && pod repo update && pod install --repo-update && cd ..`
